addons/mail/static/tests/suggestion/suggestion.test.js

```diff
@@ -314,3 +314,19 @@ test("Current user that is a follower should be considered as such", async () =>
         before: [".o-mail-Composer-suggestion", { text: "Person A(a@test.com)" }],
     });
 });
+
+test("Mention everybody", async () => {
```

Comment:
`Mention with @everyone` _(wording it that way makes is clearer what this test is about)_